Buttons.d

```diff
@@ -341,35 +343,9 @@ func void Buttons_Do() {
 		};
 		View_MoveToPxl(_BUTTON_MO, x, y);
 	};
-	var int i; i = 0;
-	var int pos; pos = MEM_StackPos.position;
-		if (i >= _Buttons_NextSlot) {
-			return;
-		};
-		
-		btn = get(MEM_ReadStatArr(_Buttons, i));
-		var int CY; CY = Print_ToVirtual(CURSOR_Y, PS_Y);
-		var int CX; CX = Print_ToVirtual(CURSOR_X, PS_X);
-		if (btn.state & BUTTON_ACTIVE) {
-			if (btn.posx <= CX && btn.posx2 >= CX && btn.posy <= CY && btn.posy2 >= CY) {
-				if (Cursor_Left==KEY_PRESSED) {
-					MEM_PushIntParam(MEM_ReadStatArr(_Buttons, i));
-					MEM_CallByID(btn.on_click);
-				};
-				if ((btn.state & BUTTON_ENTERED)==0) {
-					MEM_PushIntParam(MEM_ReadStatArr(_Buttons, i));
-					MEM_CallByID(btn.on_enter);
-					btn.state = btn.state | BUTTON_ENTERED;
-				};
-			} else if (btn.state & BUTTON_ENTERED) {
-				MEM_PushIntParam(MEM_ReadStatArr(_Buttons, i));
-				MEM_CallByID(btn.on_leave);
-				btn.state = btn.state & ~BUTTON_ENTERED;
-			}; 
-		};
-				
-		i += 1;
-		MEM_StackPos.position = pos;
+
+	foreachHndl( _Button@, _Button_processMouseEvents );
```

Comment:
- Adjust function name (see [comment in line 308](https://github.com/Lehona/LeGo/pull/18/files#diff-4a3ff6edbf0e7ae29312af1298113f4dR308))
- Remove white space between parentheses for consistent/readable code

```D
	foreachHndl(_Button@, _Button_Do_Sub);
```